site/lib/src/components/copy_button.dart

```diff
@@ -50,11 +62,9 @@ class _CopyButtonState extends State<CopyButton> {
         ...component.classes,
       ],
       title: component.title ?? 'Copy ${component.toCopy} to your clipboard.',
-      content: component.buttonText,
+      content: _copied ? 'Copied!' : component.buttonText,
```

Comment:
I **love** this solution! Works well for all usages of this component without any new styles needed.